app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -279,6 +281,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         loadStateFromBundle(savedInstanceState);
 
+        // Need to override CalendarUtil's month localizer
+        CalendarUtils.setArrayDataSource(new AndroidArrayDataSource(this));
```

Comment:
There's a single place in the app we use for a bunch of these, isn't there? Would be nice to not leave it somewhere so central.